specs/WebResourceRequested-CustomScheme.md

```diff
@@ -271,45 +275,45 @@ CHECK_FAILURE(m_webView->ExecuteScript(
 
 ```c#
 // This is the ICoreWebView2CustomSchemeRegistration interface
+// This represents the registration of a custom scheme with the
+// CoreWebView2Environment.
+// This allows the WebView2 app to be able to handle
+// WebResourceRequested event for requests with the specified scheme and
+// be able to navigate the WebView2 to the custom scheme. Once the environment
+// is created, the registrations are valid and immutable throughout the
+// lifetime of the associated WebView2s' browser process and any WebView2
+// environments sharing the browser process must be created with identical
+// custom scheme registrations, otherwise the environment creation will fail.
```

Comment:
Is order significant when checking for "identical"-ness?

Reply:
It does not. Let's link to the [CreateCoreWebView2Environment docs](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/webview2-idl?view=webview2-1.0.1210.39#createcorewebview2environmentwithoptions) which should talk about this in more detail. But don't seem to, so we should also update those docs to note that ordering doesn't matter and so on.

Reply:
I've just noted here as I don't think there is anything else in EnvironmentOptions with a list property right now and order could matter in another list property in the future.